contact.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Contact us - If Else Network</title>
+		<link rel="stylesheet" href="main.css?v=1" />
```

Comment:
When this file is renamed, this stylesheet won't be able to be found, because the path is relative. Use `/main.css` instead of `main.css`.